test_fakeredis.py

```diff
@@ -53,7 +52,7 @@ def redis2_only(func):
     @wraps(func)
     def wrapper(*args, **kwargs):
         if REDIS3:
-            raise SkipTest("Test is only applicable to redis-py 2.x")
+            pytest.skip("Test is only applicable to redis-py 2.x")
         return func(*args, **kwargs)
```

Comment:
This'll work, but I think it should be possible to simplify it a lot using [skipif](https://doc.pytest.org/en/latest/skipping.html#id1) - something like (untested)
```python
redis2_only = pytest.mark.skipif(REDIS3, "Test is only applicable to redis-py 2.x")
```